<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>affintransformation_gui</class>
 <widget class="QDialog" name="affintransformation_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Affintransformation</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Drehwinkel Y-Achse (&amp;#945;) </string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;transformierte Punkte&lt;/b&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_alpha">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_a2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_transY">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_berechne">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>berechne</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Translation Y&lt;sub&gt;0&lt;/sub&gt; (m)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dateipfad">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_my">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_test2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Testdatei 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a&lt;sub&gt;1&lt;/sub&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_out">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>261</width>
     <height>731</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Maßstab Y-Achse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a&lt;sub&gt;2&lt;/sub&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_a1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_transX">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Translation X&lt;sub&gt;0&lt;/sub&gt; (m)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_in">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>59</y>
     <width>261</width>
     <height>731</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;Transformationsparamenter:&lt;/b&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_affin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_speichern">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>tansformierte Punkte als JSON - Datei speichern</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_datei">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>json-Datei laden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_test">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Testdatei 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;Eingangsdatei&lt;/b&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Maßstab X-Achse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mx">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_beta">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Drehwinkel Y-Achse (&amp;#946;) </string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_a3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a&lt;sub&gt;4&lt;/sub&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_a4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a&lt;sub&gt;3&lt;/sub&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_rstklf">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>60</y>
     <width>261</width>
     <height>731</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>30</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Restklaffen identischer Punkte&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
